src/utils/date.ts

```diff
@@ -0,0 +1,36 @@
+import dayjs from "dayjs";
+import "dayjs/locale/sk";
+const TODAY = 0;
+const TOMMOROW = 1;
```

Comment:
if we don't need TODAY and TOMORROW in other functions, we could declare them in the `getCalendar` function

Reply:
also can you put an empty line before the `getCalendar` function

Reply:
Peter's decision to put the constants before function declaration, but I'll move them to the function